Rework the introductory content (#2193)
buildkite/docs: pages/tutorials/getting_started.md

The following diagram shows the split in Buildkite between the SaaS platform and the agents running on your infrastructure.

<%= image "hybrid-saas-architecture.svg", alt: "Shows the hybrid architecture combining a SaaS platform with your infrastructure" %>

The diagram shows that Buildkite provides a web interface, handles integrations with third-party tools, and offers APIs and webhooks. By design, sensitive data, such as source code and secrets, remain within your environment and are not seen by Buildkite. This decoupling provides flexibility and security as you maintain control over the build environment and agent scaling while Buildkite manages the coordination, scheduling, and web interface.

## Install and run an agent

The program that executes work is called an _agent_ in Buildkite. An agent is a small, reliable, and cross-platform build runner that connects your infrastructure to Buildkite. It polls Buildkite for work, runs jobs, and reports results. You can install agents on local machines, cloud servers, or other remote machines. You need at least one agent to run builds.

>📘 Already running an agent

> > If you're already running an agent, skip this section.

Reviewer: Can we link this directly to the next relevant section?

Author: Good idea!

```suggestion
> If you're already running an agent, skip to the [next step](#create-a-pipeline).
```
